cmd/simulator/load/awm.go

```diff
@@ -0,0 +1,379 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package load
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"errors"
+	"fmt"
+	"math/big"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils/crypto/bls"
+	"github.com/ava-labs/avalanchego/utils/set"
+	"github.com/ava-labs/avalanchego/utils/wrappers"
+	avalancheWarp "github.com/ava-labs/avalanchego/vms/platformvm/warp"
+	"github.com/ava-labs/subnet-evm/cmd/simulator/txs"
+	"github.com/ava-labs/subnet-evm/core/types"
+	"github.com/ava-labs/subnet-evm/ethclient"
+	"github.com/ava-labs/subnet-evm/interfaces"
+	"github.com/ava-labs/subnet-evm/params"
+	"github.com/ava-labs/subnet-evm/tests/utils/runner"
+	predicateutils "github.com/ava-labs/subnet-evm/utils/predicate"
+	warpclient "github.com/ava-labs/subnet-evm/warp"
+	"github.com/ava-labs/subnet-evm/warp/payload"
+	"github.com/ava-labs/subnet-evm/x/warp"
+	"github.com/ethereum/go-ethereum/common"
+	ethcrypto "github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/log"
+	"golang.org/x/sync/errgroup"
+)
+
+func MkSendWarpTxGenerator(chainID *big.Int, dstChainID ids.ID, gasFeeCap, gasTipCap *big.Int) txs.CreateTx[*AwmTx] {
+	txGenerator := func(key *ecdsa.PrivateKey, nonce uint64) (*AwmTx, error) {
+		addr := ethcrypto.PubkeyToAddress(key.PublicKey)
+		input := warp.SendWarpMessageInput{
+			DestinationChainID: common.Hash(dstChainID),
+			DestinationAddress: addr,
+			Payload:            getTestWarpPayload(dstChainID, addr, nonce),
+		}
+		packedInput, err := warp.PackSendWarpMessage(input)
+		if err != nil {
+			return nil, err
+		}
+		signer := types.LatestSignerForChainID(chainID)
+		tx, err := types.SignNewTx(key, signer, &types.DynamicFeeTx{
+			ChainID:   chainID,
+			Nonce:     nonce,
+			To:        &warp.Module.Address,
+			Gas:       200_000,
+			GasFeeCap: gasFeeCap,
```

Comment:
constant?

Reply:
I'll add a comment that this is just sufficient gas, I don't think we need to have a constant for every arbitrary gas amount.